src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.2_1.cs

```diff
@@ -1,8 +1,8 @@
 #if MESSAGEPACK_2_1
 using System;
 using Datadog.Trace.ExtensionMethods;
-using MessagePack;
-using MessagePack.Formatters;
+using Datadog.Trace.Vendors.MessagePack;
```

Comment:
We can probably delete this entire file. It's not used now and we'll write a custom serializer before we ever switch to MessagePack 2.1.

Reply:
I removed this file and the MESSAGEPACK compilation flags in 608ccad6c14264efbbfb8f046c155094d789aa02